en/04_Changelogs/5.0.0.md

```diff
@@ -82,6 +82,11 @@ This is a major release and as a result there are a number of breaking API chang
 
 ### General changes {#api-general}
 
+- [SecurityAdmin](api:SilverStripe\Admin\SecurityAdmin) now extends [ModelAdmin](api:SilverStripe\Admin\ModelAdmin) instead of [LeftAndMain](api:SilverStripe\Admin\LeftAndMain) so that it can make use of standard API. This was done to remove duplicate code.
+  - `MemberImportForm.php` has been removed
+  - `GroupImportForm.php` has been removed
+  - Previous extension hook implementations of `SecurityAdmin::updateEditForm($form)` should still work as the extension hook is called in `ModelAdmin::getEditForm()`
```

Comment:
The actual form is slightly different so the extensions may need to be reworked slightly to work with the way modeladmin's form is produced.... but we can work out the exact wording for that closer to release. For now this is good enough.